src/data-dictionary/events/ContainerSample/memorySwapOnlyUsageBytes.md

```diff
@@ -0,0 +1,9 @@
+---
+name: memorySwapOnlyUsageBytes
+type: attribute
+units: bytes (B)
+events:
+  - ContainerSample
+---
+
+The amount of swap memory the container is using. This memory doesn't include non-swap memory.
```

Comment:
```suggestion
The amount of swap memory the container is using. This metric does not include non-swap memory.
```

Reply:
we are okay with contractions, just a note